src/Features/Core/Portable/RemoveAsyncModifier/AbstractRemoveAsyncModifierCodeFixProvider.cs

```diff
@@ -0,0 +1,236 @@
+﻿// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+using System;
+using System.Collections.Immutable;
+using System.Linq;
+using System.Threading;
+using System.Threading.Tasks;
+using Microsoft.CodeAnalysis;
+using Microsoft.CodeAnalysis.CodeActions;
+using Microsoft.CodeAnalysis.CodeFixes;
+using Microsoft.CodeAnalysis.Editing;
+using Microsoft.CodeAnalysis.Shared.Extensions;
+using Roslyn.Utilities;
+using KnownTypes = Microsoft.CodeAnalysis.MakeMethodAsynchronous.AbstractMakeMethodAsynchronousCodeFixProvider.KnownTypes;
+
+namespace Microsoft.CodeAnalysis.RemoveAsyncModifier
+{
```

Comment:
```suggestion
namespace Microsoft.CodeAnalysis.RemoveAsyncModifier
{
    using KnownTypes = AbstractMakeMethodAsynchronousCodeFixProvider.KnownTypes;

```

Reply:
As above, this won't compile. I think I'll remove the usage of KnownTypes anyway, as this only needs two of them, and they're both available as extension methods on `Compilation` anyway.